Create Amber Network Documentation (#252)
oasislabs/docs: src/operators/amber-network.md

# The Oasis Amber Network

## What is it

With the [QuestNet](./the-quest-rules.md) reaching a point of stability, the
Amber Network is the next phase of the Oasis Network’s path to Mainnet.
The Oasis Amber Network is a release candidate for Mainnet and
includes all the core features of the network you can expect
at Mainnet launch.  This includes:

* Fully decentralized consensus layer
* Staking and delegation
* Commission management for node operators
* Consensus committee with up to 70 validators
* Multiple ParaTimes (parallel runtimes) running compute on

> the network (expect more and more added over time)

Reviewer: ```suggestion
  the network (expect more and more added over time)
```

## Timing

The Amber Network will kick off in mid-June. We will share the specific start date
on June 11.

## Goal of Amber
This network is intended primarily for node operators
who won tokens in The Quest as a means to do a final test of the
Oasis Network code before the Mainnet launch. Very small grants
will also be awarded to individuals who want to test hardware wallets.

## Incentives for Joining

The Amber Network will be used as a way to provide bonuses in the
form of delegation from the Oasis Protocol Foundation in the forthcoming Mainnet.
The Oasis Protocol Foundation will plan to delegate up to 50% of its available
tokens based on successful completion and staking reward rankings from the Oasis
Amber Network as well as availability rankings and overall track record.

> **(NOTE: Delegation terms to be determined with the
> Oasis Protocol Foundation at the end of the competition.)**

Reviewer: ```suggestion
::: tip NOTE: 
Delegation terms to be determined with the Oasis Protocol Foundation 
at the end of the competition.
:::
```

## Joining the Amber Network as a node operator

In order to join the Amber Network, you will need AMBER test tokens.

> *(Note: AMBER tokens are test tokens used specifically for this network).*

Reviewer: I'd like to use the `::: tip NOTE` section but this is odd to break up.

Author: I think that'd be okay.

Incentives for the Amber Network will be based on this community-proposed
incentive model and include a few parameters specific to this Network.

## Submitting your entity for the genesis block

To ensure that your node is in the genesis block, we will ask that you
submit a new entity package no later than Monday, June 15.
Note that even if you’ve submitted an entity package in the past,
you need to regenerate and submit a new one.

> Entity package instructions can be found [here](docs.oasis.dev).

Reviewer 2: TODO: update this link to point to entity package instructions

Author: @ravenac95 can you add?

## Submitting your entity for the genesis block

To ensure that your node is in the genesis block, we will ask that you
submit a new entity package no later than Monday, June 15.
Note that even if you’ve submitted an entity package in the past,
you need to regenerate and submit a new one.

> Entity package instructions can be found [here](docs.oasis.dev).

Reviewer: ```suggestion
Entity package instructions can be found [here](./creating-an-entity-package.md).
```

10% of the weighted median network commission rate)**

## Wallet testers needed

More on this to come soon!

**Disclaimer: The Amber Network and all associated activities and rewards
are not available to all participants and eligibility requirements apply.

> For details, see the Program Terms available here. This article does not

Reviewer 2: TODO: link to program terms

> **Disclaimer: The Amber Network and all associated activities and rewards
> are not available to all participants and eligibility requirements apply.
> For details, see the Program Terms available [here](https://docsend.com/view/zv5cfia).
> This article does not guarantee anyone the right to participate in the
> Amber Network or to receive rewards.**

Reviewer: ```suggestion
::: warning DISCLAIMER 
The Amber Network and all associated activities and rewards
are not available to all participants and eligibility requirements apply.
For details, see the Program Terms available [here](https://docsend.com/view/zv5cfia).
This article does not guarantee anyone the right to participate in the
Amber Network or to receive rewards.
:::
```

# The Oasis Amber Network

## What is it

With the [QuestNet](./the-quest-rules.md) reaching a point of stability, the
Amber Network is the next phase of the Oasis Network’s path to Mainnet.
The Oasis Amber Network is a release candidate for Mainnet and
includes all the core features of the network you can expect
at Mainnet launch.  This includes:

* Fully decentralized consensus layer
* Staking and delegation
* Commission management for node operators
* Consensus committee with up to 70 validators
* Multiple ParaTimes (parallel runtimes) running compute on
  the network (expect more and more added over time)

## Timing

The Amber Network will kick off in mid-June. We will share the specific start date
on June 11.

## Goal of Amber

This network is intended primarily for node operators
who won tokens in The Quest as a means to do a final test of the
Oasis Network code before the Mainnet launch. Very small grants
will also be awarded to individuals who want to test hardware wallets.

> ## Incentives for Joining

Reviewer 2: ```suggestion
## Incentives for joining
```

The Amber Network will be used as a way to provide bonuses in the
form of delegation from the Oasis Protocol Foundation in the forthcoming Mainnet.
The Oasis Protocol Foundation will plan to delegate up to 50% of its available
tokens based on successful completion and staking reward rankings from the Oasis
Amber Network as well as availability rankings and overall track record.

> **(NOTE: Initial delegation will be for at least 6 month and is contingent on

Reviewer 2: ```suggestion
**(NOTE: Initial delegation will be for at least 6 months and is contingent on
```

commission rates that are +/- 10% of the weighted median network commission

> rate and no greater than 30% with high uptime performance.)**

Reviewer 2: ```suggestion
rate and no greater than 30%, and high uptime performance.)**
```

## Joining the Amber Network as a node operator

In order to join the Amber Network, you will need AMBER test tokens.
::: Tip *Note: AMBER tokens are test tokens used specifically for this network).*
Unlike in the QuestNet, there is no faucet to distribute tokens, so instead you
can get stake needed to run your node in two ways:

* **Genesis Block**: In the genesis block of the Amber Network, node operators
will be allotted the total amount of tokens they’ve previously earned from
the Oasis QuestNet or via other means (fulfilled grants, etc). You will

> be able to find that amount here after The Quest ends on June 4.

Reviewer 2: Needs a link to quest winners?

* **Delegation** from the Oasis Protocol Foundation and Oasis Labs
(primarily for testing purposes)

Incentives for the Amber Network will be based on this community-proposed
incentive model and include a few parameters specific to this Network.

## Submitting your entity for the genesis block

To ensure that your node is in the genesis block, we will ask that you
submit a new entity package no later than Monday, June 15.
Note that even if you’ve submitted an entity package in the past,
you need to regenerate and submit a new one.

Entity package instructions can be found [here](./creating-an-entity-package.md).

> ## Wallet-testers needed

Reviewer 2: ```suggestion
## Wallet testers needed
```
